### DISCUSSION TITLE
Revise Package

### DISCUSSION THREAD

**Discussion by joseph-gergaud** [control-toolbox/control-toolbox.github.io#16] [General]
When you change something in a package it's possible not to restart julia :

https://techytok.com/lesson-workflow/

https://timholy.github.io/Revise.jl/stable/

**Comment by jbcaillau** [comment]


**Reply by jbcaillau** [reply]
Mmm... as it is not possible to "unload" a package / module, `Revise.jl` cannot do anything but issue a warning
```julia
WARNING: replacing module Foo.
WARNING: using Foo.@foo in module Main conflicts with an existing identifier.
```
